test/files/formatter/preferred-citation-masters-thesis-no-type.cff

```diff
@@ -0,0 +1,32 @@
+# A PhD thesis preferred citation
```

Comment:
This comment doesn't match the title of the file or the rest of the contents. Please can you fix?

Reply:
Done!